src/forms/staff/forms.jsx

```diff
@@ -0,0 +1,126 @@
+              </FormControl>
+              <FormMessage />
+            </FormItem>
+          )}
+        />
+        <FormField
+          control={form.control}
+          name="email"
+          render={({ field }) => (
+            <FormItem className="flex flex-col items-start">
+              <FormLabel className="text-base font-large">Email</FormLabel>
+              <FormControl>
+                <Input placeholder="email@example.com" {...field} />
+              </FormControl>
+              <FormMessage />
+            </FormItem>
+          )}
+        />
+        <FormField
+          control={form.control}
+          name="specialty"
+          render={({ field }) => (
+            <FormItem className="flex flex-col items-start w-full">
+              <FormLabel className="text-base font-large">Specialty</FormLabel>
+              <FormControl className="w-full">
+                <DropdownMenu>
+                  <DropdownMenuTrigger className="w-full text-left">
+                    <p className="flex h-9 w-full rounded-md border border-input bg-transparent px-3 py-1 text-base shadow-sm transition-colors file:border-0 file:bg-transparent file:text-sm file:font-medium file:text-foreground placeholder:text-muted-foreground focus-visible:outline-none focus-visible:ring-1 focus-visible:ring-ring disabled:cursor-not-allowed disabled:opacity-50 md:text-sm">
+                      {specialtiesWithLabels.find((s) => s.value === field.value)?.label}
+                    </p>
+                  </DropdownMenuTrigger>
+                  <DropdownMenuContent className="w-full">
+                    {specialtiesWithLabels.map((specialty) => (
+                      <DropdownMenuItem
+                        key={specialty.value}
+                        onSelect={() => field.onChange(specialty.value)}
+                      >
+                        {specialty.label}
+                      </DropdownMenuItem>
+                    ))}
+                  </DropdownMenuContent>
+                </DropdownMenu>
+              </FormControl>
+              <FormMessage />
+            </FormItem>
+          )}
+        />
+        <FormField
+          control={form.control}
+          name="password"
+          render={({ field }) => (
+            <FormItem className="flex flex-col items-start">
+              <FormLabel className="text-base font-large">Password</FormLabel>
+              <FormControl>
+                <PasswordInput {...field} />
+              </FormControl>
+              <FormMessage />
+            </FormItem>
+          )}
+        />
+        <Button className="w-full" type="submit" disabled={isLoading}>
+          {isLoading && (
+            <LoaderCircle className="mr-2 h-4 w-4 animate-spin" />
+          )}
+          <span className="text-base font-large">Register</span>
+        </Button>
+        {error && <FormMessage className="flex flex-col items-start text-base">{error}</FormMessage>}
```

Comment:
To cover all the form when using grids:

```suggestion
        <Button className="w-full lg:col-span-2" type="submit" disabled={isLoading}>
          {isLoading && (
            <LoaderCircle className="mr-2 h-4 w-4 animate-spin" />
          )}
          <span className="text-base font-large">Register</span>
        </Button>
        {error && <FormMessage className="flex flex-col items-start text-base lg:col-span-2">{error}</FormMessage>}
```